content/04.model-complexity.regularization.md

```diff
@@ -0,0 +1,55 @@
+## Regularization {.page_break_before}
+
+Machine learning algorithms are optimized to find patterns among data points. 
+Given a large amount of data, this optimization works in favor of the algorithm trying to learn complex patterns in the data. 
+However, when data is sparse, the same optimization leads to the model learning technical variances rather than actual patterns in the data. 
+This results in low bias but high variance in the prediction results i.e. overfitted models fit the training data very closely (low bias), but fail to predict new testing data with high accuracy (high variance) leading to low prediction accuracy of the model in view of new data. 
+Such overfitting can lead to misinterpretation of technical variances as true patterns in the sparse data and thus needs to be minimized. 
+Minimization of overfitting can be accomplished by crossvalidation and regularization methodologies. 
+While crossvalidation aims to reduce the variance in prediction, regularization adds a small amount of bias to the initial model to minimize its dependance and sensitivity to training data. 
+Regularization makes models less sensitive to variance in sample data by adding a penalty (lambda) to the sum of squared residuals (or sum of maximum likelihoods). 
+While the value of lambda is determined by crossvalidation, the minimization of the sum of squared residuals (or error) leads to a model that is less sensitive to changes in sample data.
+
+To learn from 'n' number of features, machine learning models ideally need at least 'n+1' samples. 
+For eg. to predict any disease outcome from 20,000 genes we need at least 20,001 samples. 
+However, in reality even the best sample set probably contains 500 samples, and in case of rare diseases that may be closer to 50 than 500. 
```

Comment:
A citation to support this would be helpful

Reply:
edited this part out, since it was mainly a conceptual approximation without any specific citation.